docker/package/package_generator.py

```diff
@@ -234,7 +232,7 @@ def gen_install(pkg: Package, out):
 for package in packages:
     if package_to_build is None or package.get_full_name() == package_to_build:
         if is_ubuntu:
-            dir = f"{package.get_full_name().lower()}-0ubuntu{version}"
+            dir = f"{package.get_full_name().lower()}-{version}"
```

Comment:
Based on what do we construct this `dir` string?

Reply:
for ubuntu, this dir is source for `orig.tar.gz` archive, so it has the same name for simplicity.

Reply:
But we changed it, didn't we? If we did, why? I see that we archive it later, so it's supposed to exist already. Where is it created?

Reply:
> If we did, why? 

Because we changed the versioning scheme, not it's `<upstream version>-0ubuntu<release number>`, so  
whole `0ubuntu<release number>` is like a release tag now, so the version of the sources is just `<upstream version>`

> Where is it created?

`os.rename(f"{opam_package}-bundle", dir)` a bit lower

Reply:
So we choose this name ourselves, right? There is no requirement like "there should be a folder with name <name-format> with all sources", right?

Reply:
well, AFAIU, for ubuntu we actually need a folder with such a name to be able to run `dh_make` in it

Reply:
AFAIU the directory was something like `tezos-node-0ubuntu7.4`, but now it is `tezos-node-7.4`. So which folder is actually required?

Reply:
we switched versioning, previously version without release info was `0ubuntu7.4`, not it is `tezos-node-7.4`, now the latter is required

Reply:
Ok